CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Allow out-of-bounds (i.e. `'0`) top end address in addr_map of `addr_decode` module for end of address space.
 - Update CI.
+- Remove asserts in `stream_fifo_optimal_wrap`
```

Comment:
This is not included in `v1.26.0`. Could you add this to a new "unreleased" section? i.e.
```
## Unreleased
### Changed
- Remove asserts ...

## 1.26.0 ...
```

Reply:
Uups, sorry. I changed it now.